### ISSUE TITLE
Sounds are not playing

### ISSUE THREAD

**Issue by Zartrix** [AstralOrdana/BetterBlockSounds#3]
Using only Fabric API, Mod menu and BBS on latest Fabric, there are none of the supposed sounds playing.
[latest.log](https://github.com/AstralOrdana/BetterBlockSounds/files/12199421/latest.log)
log says something about
```
[00:32:12] [main/WARN]: Error loading class: net/minecraft/world/level/block/Block (java.lang.ClassNotFoundException: net/minecraft/world/level/block/Block)
[00:32:12] [main/WARN]: @Mixin target net.minecraft.world.level.block.Block was not found bbs.mixins.json:SoundsMixin from mod bbs
```

**Comment by AstralOrdana** [comment]
found this on my end as well, working on a fix

**Comment by AstralOrdana** [comment]
fixed for 0.1.3
